memory/alloydb/chat_message_history.go

```diff
@@ -43,11 +43,11 @@ func NewChatMessageHistory(ctx context.Context, engine alloydbutil.PostgresEngin
 	}
 	cmh, err = applyChatMessageHistoryOptions(cmh, opts...)
 	if err != nil {
-		return ChatMessageHistory{}, fmt.Errorf("applyChatMessageHistoryOptions(): %w", err)
+		return ChatMessageHistory{}, fmt.Errorf("applyChatMessageHistoryOptions() unable to apply provided options: %w", err)
```

Comment:
nit: I don't think the method name adds value here -- the end user will not be aware of the function used at this point